src/Peasy/Peasy.Tests/ServiceBaseTests.cs

```diff
@@ -16,7 +14,7 @@ public class ServiceBaseTests
         public void Ensure_Method_Invocations_For_GetAllCommand()
         {
             var service = new ServiceBaseMock(new PersonProxyStub());
-            var result = service.GetAllCommand().Execute();
```

Comment:
Same here, I think I left for clarity for the reader.  I can be persuaded though ;)

Reply:
Same as above :)